phpunit.xml

```diff
@@ -9,6 +9,7 @@
   >
   <php>
     <ini name="error_reporting" value="-1" />
+    <env name="SYMFONY_DEPRECATIONS_HELPER" value="62" /> <!-- there are 62 known issues that can't cause CI to fail -->
```

Comment:
This might look a bit strange :) As of now in the Kernel tests there are 62 deprecation warnings. This setting is described in [Making Tests Fail](http://symfony.com/doc/current/components/phpunit_bridge.html#making-tests-fail) section of PHPUnit Bridge docs. I think the idea behind this setting is to allow passing tests with deprecated code but disallow introducing new deprecated code. Could also be set globally to `SYMFONY_DEPRECATIONS_HELPER="weak"` but a disadvantage of this is that deprecated warnings are not shown on the output.
Or maybe CI failing on deprecated stuff is a good approach and this workaround is not needed? WDYT @andrerom?